pkg/occlient/occlient.go

```diff
@@ -1246,7 +1246,7 @@ func (c *Client) clusterServiceClassExists(name string) bool {
 
 // CreateRoute creates a route object for the given service and with the given
 // labels
-func (c *Client) CreateRoute(name string, serviceName string, labels map[string]string) (*routev1.Route, error) {
+func (c *Client) CreateRoute(name string, serviceName string, portNumber intstr.IntOrString, labels map[string]string) (*routev1.Route, error) {
```

Comment:
the comment should explain what `portNumber` argument is.
It would be better to have portNumber as int  and convert it to `IntOrString` inside this function.